tests/test_api.py

```diff
@@ -147,3 +149,16 @@ def test_subsection_max_min(self, mock_graded_subsections):
         rows = list(processor.get_iterator())
         assert len(rows) != self.NUM_USERS + 1
 
+    @patch('lms.djangoapps.grades.api.CourseGradeFactory.read')
+    def test_course_grade_filters(self, course_grade_factory_mock):
+        self._make_enrollments()
+
+        course_grade_factory_mock.return_value = Mock(percent=0.50)
+
+        processor = api.GradeCSVProcessor(course_id=self.course_id, max_points=100, course_grade_min=60)
+        rows = list(processor.get_iterator())
+        self.assertNotEqual(len(rows), self.NUM_USERS+1)
```

Comment:
I think all the other assertions in this file are written as `assert` statements. For consistency we may want to conform. (I vaguely recall there being a particular subtle distinction in the behavior of the two, I think in how assertion failures are reported, in favor of the `assert` statement. But I'd have to dig to remember what that distinction was. The rule I've been working under has been to use `assert` when I write new test files, and conform when I'm adding tests to a file, since my understanding is that it's not a super material enough difference to warrant conversion.)

Reply:
Thanks for pointing that out. I think it is the other way around. https://stackoverflow.com/questions/48216473/using-asserttrue-vs-assertequal-in-unittest